packages/markdown/components/Embed.jsx

```diff
@@ -20,7 +22,7 @@ class Embed extends React.Component {
             }}
             target="_blank"
           >
-            <b style={{ color: '#333' }}>View Embed:</b> <span style={{ opacity: 0.75 }}>{this.props.url}</span>
+            <b style={{ color: '#333' }}>View Embed:</b> <span style={{ opacity: 0.75 }}>{url}</span>
```

Comment:
Any preference over using `strong`? Just curious.

Reply:
for brevity's sake, and because it matches up nicely with the single-letter `<i>` tag, which makes for nice typographic markup!